build.sbt

```diff
@@ -7,10 +7,11 @@ import bijection._
 
 val buildLevelSettings = Seq(
   organization := "com.twitter",
-  crossScalaVersions := Seq("2.10.6", "2.11.8"),
+  crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0"),
   scalaVersion := "2.11.8",
   javacOptions ++= Seq("-source", "1.6", "-target", "1.6"),
   javacOptions in doc := Seq("-source", "1.6"),
+  coverageEnabled := true,
```

Comment:
Yeah so I originally tried to do that. The `+++$TRAVIS_SCALA_VERSION coverage` command doesn't work with the sbt-doge plugin for some reason (see above): https://github.com/twitter/bijection/pull/258#issuecomment-263434782

I needed to pull in the sbt-doge plugin cause I wanted to ensure that bijection-scrooge, bijectionUtil and bijectionFinagleMySql can override the cross-scala versions to skip 2.12 as we don't have the finagle artifacts. Based on searching that seemed to be the best option to skip modules selectively for a given scala version. If you folks know of a better alternative, I could try that and drop the doge plugin.